RecoHGCal/TICL/interface/Trackster.h

```diff
@@ -0,0 +1,29 @@
+// Author: Felice Pantaleo - felice.pantaleo@cern.ch
+// Date: 09/2018
+// Copyright CERN
```

Comment:
@tommasoboccali, @klute, @dpiparo although our code is already full of "copyright" statements, I wonder about the compatibility of this lines and https://github.com/cms-sw/cmssw/blob/master/LICENSE

Reply:
I have to admit i am not really passionate with CP discussion, but indeed the top level file 

https://github.com/cms-sw/cmssw/blob/master/NOTICE

already says

Copyright 2019 CERN¹ for the benefit of the CMS Collaboration²

so it should be a catch-all for the code

Reply:
@tommasoboccali thanks, these discussions are not passionating but I have to check what we store in our repositories, now after @rovere last update there is not reason to further delay the integration of this code